react/typings/packages-sku-ids.d.ts

```diff
@@ -0,0 +1,9 @@
+export type PackagesSkuIds = {
```

Comment:
La fel ca si la minicart, ar trebui sa fie interface si sa nu il exporti de aici, si implicit sa nu il importi in celelalte fisiere.

Reply:
Also, modifica vendor, changelog, si ce mai e nevoie.